tests/testthat/test-renderer4-update-axes-multiple-ss.R

```diff
@@ -0,0 +1,312 @@
+acontext("update_axes - multiple single selectors")
+
+## Test for appropriate warnings
+set.seed(123)
+
+# Empty domains in axis updates -> generate warning
+data_f1 <- data.frame(a=runif(30, 1, 30), b=sample(1:30))
+data_f1$ss1 <- as.factor(1:3)
+data_f1$ss2 <- as.factor(c("alpha", "beta"))
+data_f1$ss3 <- as.factor(c("A", "A", "B"))
+# factors "1" & "2" are never paired with factor "B"
+# factor "3" is never paired with factor "A"
+plot1 <- ggplot() + geom_point(aes(a,b, showSelected1=ss1,
+                                   showSelected2=ss2,
+                                   showSelected3=ss3),
+                               data = data_f1) +
+  theme_animint(update_axes=c("x"))
+viz <- list(p=plot1)
+
+viz$selector.types <- list(ss1="single", ss2="single", ss3="single")
+expect_warning(animint2HTML(viz),
+               "some data subsets have no data to plot")
+
+# Only a single unique value in domains for axis updates -> generate warning
+data_f2 <- data.frame(a=runif(6, 1, 6), b=sample(1:6))
+data_f2$ss1 <- as.factor(1:3)
+data_f2$ss2 <- as.factor(c("alpha", "beta"))
+# Each factor interaction only possesses a single value
+plot2 <- ggplot() + geom_point(aes(a,b, showSelected1=ss1,
+                                   showSelected2=ss2),
+                               data = data_f2) +
+  theme_animint(update_axes=c("x"))
+viz <- list(p=plot2)
+
+viz$selector.types <- list(ss1="single", ss2="single")
+expect_warning(animint2HTML(viz),
+               "some data subsets have only a single data value to plot")
+
+# Axes updates for more than one single selectors -> no warnings
+data_f3 <- data.frame(a=runif(60, 1, 60), b=sample(1:60))
+data_f3$ss1 <- as.factor(1:3)
+data_f3$ss2 <- as.factor(c("alpha", "beta"))
+plot3 <- ggplot() + geom_point(aes(a,b, colour=ss1,
+                                   showSelected2=ss2),
+                               data = data_f3)
+  
+viz <- list(p=plot3 + theme_animint(update_axes=c("x")))
+
+viz$selector.types <- list(ss1="single", ss2="single")
+expect_no_warning(animint2HTML(viz))
+
+## --------------------------------------------------------------------- ##
+## Tests for axis updates for more than one showSelected vars
+
+# Plots with axis updates
+no_updates <- plot3
+
+update_x <- no_updates+
+  theme_animint(update_axes=c("x"))
+update_y <- no_updates+
+  theme_animint(update_axes=c("y"))
+update_xy <- no_updates+
+  theme_animint(update_axes=c("x","y"))
+
```

Comment:
@cpsievert That is certainly true. Most users tend to skip the documentation.

But I am a little hesitant in doing this as we do not support `multiple` selection vars as of now.  So users not specifying their selection vars as `single` will keep getting warnings they do not expect. This could get a bit annoying when they get warnings of the likes of "created plot with no updates" and stuff, don't you think? So I would suggest the `update_axes=TRUE` approach for now...

Maybe once we have implemented this for both types of selections and sorted out the current bugs, we could definitely do that. Any thoughts?